Doc/library/functions.rst

```diff
@@ -1800,6 +1800,13 @@ are always available.  They are listed here in alphabetical order.
    the second argument is a type, ``issubclass(type2, type)`` must be true (this
    is useful for classmethods).
 
+   When called directly within an ordinary method of a class, both arguments may
+   be omitted ("zero-argument :func:`super`"). In this case, *type* will be the
+   enclosing class, and *obj* will be the first argument of the immediately
+   enclosing function (typically ``self``). (This means that zero-argument
+   :func:`super` will not work as expected within a nested function or a generator
+   expression, which implicitly creates a nested function.)
```

Comment:
Some very minor suggested wording tweaks:
- No need to link to the function reference we're already in; use `!` to prevent Sphinx from adding links when `super` is referenced
- Some suggested changes to the last sentence to make it more elegant.

```suggestion
   When called directly within an ordinary method of a class, both arguments may
   be omitted ("zero-argument :func:`!super`"). In this case, *type* will be the
   enclosing class, and *obj* will be the first argument of the immediately
   enclosing function (typically ``self``). (This means that zero-argument
   :func:`!super` will not work as expected within nested functions, including
   generator expressions, which implicitly create nested functions.)
```

Reply:
👍